### PR TITLE
Add possibility to save velocity snapshots

### PR DESCRIPTION
**Pull request by mthieffry**
It's working with my examples.

- Need to test on other examples.

- Need to decide if "Save velocity" is an option or always true ?

### CODE REVIEW

File: python/mor/reduction/reduceModel.py [hunk 1]
```diff
@@ -262,7 +263,7 @@ def copyFileIntoAnother(self,fileToCopy,fileToPasteInto):
         except:
             raise
 
-    def copyAndCleanState(self,results,periodSaveGIE,stateFileName,gie=None):
+    def copyAndCleanState(self,results,periodSaveGIE,stateFileName,velocityFileName,gie=None):
```
Comment (VannesteFelix) [inline python/mor/reduction/reduceModel.py]: it will be better if this new argument was by default None and in this case we save nothing.
Maybe we can see also to have an argument allowing to either save state, or velocity or even both...

File: python/mor/reduction/template/phase1_snapshots.py [hunk 2]
```diff
@@ -52,22 +53,27 @@ def createScene(rootNode):
     else:
         AnimationManager(rootNode)
 
-    # Now that we have the AnimationManager & a list of the node we want to animate
+    # Now that we have the AnimationManager & a list of the nodes we want to animate
```
Comment (VannesteFelix) [inline python/mor/reduction/template/phase1_snapshots.py]: thanks for the corrections

File: python/mor/reduction/template/phase1_snapshots.py [hunk 3]
```diff
@@ -17,6 +17,7 @@
 
 # Our Original Scene IMPORT
 originalScene = '$ORIGINALSCENE'
+import os
```
Comment (VannesteFelix) [inline python/mor/reduction/template/phase1_snapshots.py]: why did you add it ?

File: python/mor/reduction/reduceModel.py (hunk 1 above)
Comment (mthieffry) [reply python/mor/reduction/reduceModel.py]: I added an option to save both position and velocity snapshots, or save only position. It's an "intermediate" solution, as velocity vectors are always saved (=> write state file for velocity is always added in the scene).
But the velocity state file is only built if the option is set to true.

**Comment by damienmarchal** [comment]
Dear all,

Is the PR ready or to be closed ? 

Damien.

**Comment by marques-bruno** [comment]
[ci-build]

**Comment by marques-bruno** [comment]
@VannesteFelix @mthieffry This PR has conflicts. could you take a look while I fix the CI?

**Comment by marques-bruno** [comment]
@VannesteFelix the CI *should* be back to normal, so as soon as you fix the conflict, it should be all set ;)
